.zuul.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- job:
+    name: molecule
+    nodeset: ansible-cloud-centos-9-stream-tiny
```

Comment:
ansible-cloud-centos-9-stream-tiny is a label. From label, we need to define the nodeset.
```
    nodeset:
      nodes:
        name: primary
        label: ansible-cloud-centos-9-stream-tiny
```
Can we try with this?